staff/daniel-toledo/cv/index.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>CV Daniel Toledo</title>
+</head>
+<body>
+    <header>
+        <h1>Curriculum Vitae Daniel Toledo</h1>
+        <h3>Renewable Energy Engineer and Future Javascript Programmer</h3>
+        <p>30th April 1992</p>
+        <p>Santa Catalina 18 1r, 08014 Barcelona, Spain</p>
+        <p><b>Tel :</b> +34 692461556</p>
```

Comment:
The <b> element is used to define bold (text). This is to style it.
According to the HTML 5 specification, the <b> tag should be used as a LAST resort when no other tag is more appropriate. The HTML 5 specification states that headings should be denoted with the <h1> to <h6> tags, emphasized text should be denoted with the <em> tag, important text should be denoted with the <strong> tag, and marked/highlighted text should use the <mark> tag.